recipes-support/safeboot/safeboot_git.bb

```diff
@@ -0,0 +1,80 @@
+DESCRIPTION = "Safe Boot: Booting Linux Safely"
+HOMEPAGE = "https://github.com/osresearch/safeboot"
+
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/git/LICENSE.txt;md5=b234ee4d69f5fce4486a80fdaf4a4263"
+
+SRC_URI = "git://github.com/osresearch/safeboot.git;protocol=https;branch=master;name=safeboot"
+SRC_URI += "gitsm://github.com/osresearch/sbsigntools.git;protocol=https;destsuffix=git/sbsigntools;name=sbsigntools"
+SRC_URI += "gitsm://git.kernel.org/pub/scm/linux/kernel/git/jejb/efitools.git;destsuffix=git/efitools;protocol=https;name=efitools"
+SRC_URI += "gitsm://github.com/tpm2-software/tpm2-tools.git;destsuffix=git/tpm2-tools;protocol=https;name=tpm2-tools"
+SRC_URI += "gitsm://github.com/osresearch/tpm2-totp.git;destsuffix=git/tpm2-totp;branch=static-link;protocol=https;name=tpm2-totp"
+SRC_URI += "gitsm://github.com/tpm2-software/tpm2-tss.git;destsuffix=git/tpm2-tss;protocol=https;name=tpm2-tss"
```

Comment:
Is there a point of cloning repos other than the main one if we do not compile anything?

Reply:
Removed